tests/robot-tests/tests/libs/create_emulator_release_files.py

```diff
@@ -178,21 +187,32 @@ def create_private_release_files(self):
             container_client = self.blob_service_client.get_container_client(
                 "releases")
 
-        self.upload_files(container_client,
-                          self.data_files)
-        self.upload_files(container_client,
-                          self.metadata_files)
+        try:
+            self.upload_files(container_client,
+                              self.data_files)
+        except ResourceExistsError:
+            pass
 
-    def upload_files(self, container_client, files):
-        data = b'abcd'*128
+        try:
+            self.upload_files(container_client,
+                              self.metadata_files)
+        except ResourceExistsError:
+            pass
+
+    @staticmethod
+    def upload_files(container_client, files):
+        data = b'abcd' * 128
         for file in files:
             path = file.path()
-            blob_client = container_client.get_blob_client(path)
-            blob_client.upload_blob(
-                data, blob_type="BlockBlob", metadata=file.metadata())
+            try:
+                blob_client = container_client.get_blob_client(path)
+
+                blob_client.upload_blob(
+                    data, blob_type="BlockBlob", metadata=file.metadata())
+            except ResourceExistsError:
+                pass
 
 
-if __name__ == '__main__':
-    generator = ReleaseFilesGenerator()
-    generator.create_public_release_files()
-    generator.create_private_release_files()
+generator = ReleaseFilesGenerator()
+generator.create_public_release_files()
+generator.create_private_release_files()
```

Comment:
Does this mean that just importing the file will run the generator methods? We should probably be executing these in `run_tests.py` instead